k8s/helmfile/env/production/ingress-nginx.values.yaml.gotmpl

```diff
@@ -11,10 +11,10 @@ controller:
       cpu: null
   config:
     http-snippet: >-
-      limit_req_zone $host zone=limit_host:10m rate=10r/s;
-      limit_req zone=limit_host burst=20 delay=10;
+      limit_req_zone $host zone=limit_host:10m rate=15r/s;
+      limit_req zone=limit_host burst=40 delay=10;
```

Comment:
looks good; from a chat I had with @addshore this rather than the source address seemed to be what was impacting him